tests/test_routes.py

```diff
@@ -0,0 +1,247 @@
+import pytest
+from werkzeug.exceptions import HTTPException
+from app.routes_helpers import validate_model
+from app.models.planet import Planet
+
+def test_get_all_planets_with_no_records(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert len(response_body) == 0
+    assert response_body == []
+
+
+def test_get_all_planets_with_two_records(client, two_saved_planets):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert len(response_body) == 2
+    assert response_body[0] == {
```

Comment:
Unless we have some explicit ordering behavior in our endpoints, it can be fragile to assume the position of records in the result. This could be made predictable by including an `order_by` in the get all endpoint, or by packing the result list into a dictionary of id→record.